src/Controls/src/Build.Tasks/nuget/buildTransitive/netstandard2.0/Microsoft.Maui.Controls.targets

```diff
@@ -269,7 +269,10 @@
                                       Condition="'$(MauiHybridWebViewSupported)' != ''"
                                       Value="$(MauiHybridWebViewSupported)"
                                       Trim="true" />
+      <RuntimeHostConfigurationOption Include="Microsoft.Maui.RuntimeFeature.EnableDiagnostics"
+                                      Condition="'$(EnableDiagnostics)' != ''"
+                                      Value="$(EnableDiagnostics)"
+                                      Trim="true" />
```

Comment:
I wonder if we should introduce `$(EnableMauiDiagnostics)`, and its default value is based on `$(EnableDiagnostics)`?

That way you could turn them off/on independently if needed.